editor.planx.uk/src/@planx/components/Checklist/Editor/Editor.test.tsx

```
@@ -0,0 +1,67 @@
+import { screen, waitFor } from "@testing-library/react";
+// eslint-disable-next-line no-restricted-imports
+import userEvent, {
+  PointerEventsCheckLevel,
+} from "@testing-library/user-event";
+import React from "react";
+import { DndProvider } from "react-dnd";
+import { HTML5Backend } from "react-dnd-html5-backend";
+import { setup } from "testUtils";
+
+import { ChecklistEditor } from "./Editor";
+
+describe("Checklist editor component", () => {
+  it("renders without error", () => {
+    setup(
+      <DndProvider backend={HTML5Backend}>
+        <ChecklistEditor text={""} />
+      </DndProvider>,
+    );
+    expect(screen.getByText("Checklist")).toBeInTheDocument();
+    expect(screen.getByText("add new option")).toBeInTheDocument();
+  });
+
+  it("displays the grouped checklist inputs when the 'expandable' toggle is clicked", async () => {
+    const { user } = setup(
+      <DndProvider backend={HTML5Backend}>
+        <ChecklistEditor text={""} />
+      </DndProvider>,
+    );
+
+    await waitFor(() => user.click(screen.getByLabelText("Expandable")));
+
+    const groupedOptionsEditor = await screen.findByPlaceholderText(
+      "Section Title",
+    );
+    expect(groupedOptionsEditor).toBeInTheDocument();
+  });
+
+  it.todo(
```

Appreciate groundwork for `Editor`-side test coverage here! 

I wonder if the click issue you describe is perhaps a role-based permissions error failing silently/lazily (eg view-only modal but we haven't actually disabled buttons etc)? Have you tried to additionally mock a user with editing privileges like here https://github.com/theopensystemslab/planx-new/blob/main/editor.planx.uk/src/%40planx/components/FileUploadAndLabel/Editor.test.tsx ?

---

Thanks, that was totally the issue! 😍 Have added that `beforeEach` now, and it fixed the click problem 🌟